src/Tribe/Integrations/Manager.php

```diff
@@ -168,4 +170,17 @@ private function load_beaver_builder_integration() {
 
 		return true;
 	}
+
+	/**
+	 * Loads the Fusion integration if Fusion Core is currently active.
+	 *
+	 * @since TBD
+	 */
+	public function load_fusion_integration() {
+		if ( ! defined( 'FUSION_CORE_VERSION' ) || empty( FUSION_CORE_VERSION ) ) {
+			return;
+		}
```

Comment:
Maybe a personal taste but I would move all conditionals into the service provider. Like this one: 

- https://github.com/the-events-calendar/the-events-calendar/blob/7104d6f80d1c3a9d5ca3b7e07ea49a584717288f/src/Tribe/Integrations/Fusion/Service_Provider.php#L28